externs/olx.js

```diff
@@ -2333,6 +2333,15 @@ olx.format.WFSWriteTransactionOptions.prototype.handle;
 
 
 /**
+ * Must be set to true if the transaction is for a 3D layer. This will allow
+ * the Z coordinate to be include in the transaction.
+ * @type {boolean|undefined}
+ * @api
+ */
+olx.format.WFSWriteTransactionOptions.prototype.is3D;
```

Comment:
> I wonder if hasZ is a better name since we are using that already in the library?

Good question. I have no strong opinion on this. If it's already used, I say I should change is3D to hasZ.